AzTS Data Table/Readme.md

```diff
@@ -0,0 +1,273 @@
+# AzTS Data Table
```

Comment:
Can you check the formatting of content in this file?
Pretty inconsistent
Some extra spaces in between words, JSONs have extra spaces in between, stuff like that maybe needs a quick polishing

Reply:
I have formatted where required. You can see the preview of this file. Everything looks good to me.